index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube page by Adorer</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class = "flex-div">
+        <div class="nav-left flex-div">
+            <img src="images/menu.png" alt="menu-icon" class="menu-icon">
+            <img src="images/logo.png" alt="youtube icon" class="logo-icon">
+        </div>
+
+        <div class="nav-center flex-div">
+            <div class="search-box flex-div">
+                <input type="text" placeholder="Search">
+                <img src="images/search.png" alt="search-icon">
+             </div>
+             <img src="images/voice-search.png" alt="voice-icon" class = "voice-icon">
+        </div>
+
+        <div class="nav-right flex-div" >
+            <img src="images/upload.png" alt="upload-icon">
+            <img src="images/more.png" alt="images-icon">
+            <img src="images/notification.png" alt="notification-icon">
+            <img src="images/Jack.png" alt="menu icon" class = "user-icon">
+        </div>
+    </nav>
+
+    <!-- sidebar -->
+    <div class="sidebar">
+        <div class="shortcut-links">
+            <a href=""><img src="images/home.png" alt="home-icon"><p>Home</p></a>
+            <a href=""><img src="images/explore.png" alt="explore-icon"><p>Explore</p></a>
+            <a href=""><img src="images/subscriprion.png" alt="subscriprion-icon"><p>Subscription</p></a>
+            <a href=""><img src="images/library.png" alt="library-icon"><p>Library</p></a>
+            <a href=""><img src="images/history.png" alt="history-icon"><p>History</p></a>
+            <a href=""><img src="images/playlist.png" alt="playlist-icon"><p>Playlist</p></a>
+            <a href=""><img src="images/messages.png" alt="messages-icon"><p>Messages</p></a>
+            <a href=""><img src="images/show-more.png" alt="show-more-icon"><p>Show-more</p></a>
+            <hr>
+        </div>
```

Comment:
I think the enclosing your 'a' tag in 'ul' tag here will make your code look better and more readable. 👍